openbb_terminal/cryptocurrency/onchain/topledger_model.py

```diff
@@ -0,0 +1,102 @@
+"""Topledger model"""
+__docformat__ = "numpy"
+
+import logging
+from typing import Any, Optional, Tuple
+
+import pandas as pd
+
+from openbb_terminal.decorators import log_start_end
+from openbb_terminal.helper_funcs import request
+from openbb_terminal.rich_config import console
+
+logger = logging.getLogger(__name__)
+
+MAPPING = {
+    "tensor": {
+        "api_key": "8eSjJnSuOC7uLWkrTQoWADStkNS2ZIy94pH5CsNn",
```

Comment:
this api key is open to everyone right now, i.e., everyone will be able to see it. is that ok for you? 

in one hand this would be good because users could use the command straightaway without the need to set up extra keys

Reply:
we also have the possibility to allow users to set the "TOPLEDGER_KEY" from the `keys` menu if you prefer

Reply:
these are some of the publicly available data points,
hence we have chosen to put the the respective API keys as part of code itself, enabling users to quickly get started without the need of setting up any api keys from our platform.